package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "dev": "nodemon node_modules/.bin/babel-node index.js",
-    "start": "node_modules/.bin/babel-node index.js",
+    "build": "babel ./ -d dist --copy-files --ignore node_modules",
```

Comment:
This line needed to ignore node_modules with `\"**node_modules\*` in order to work on Heroku.

Reply:
I didnt get it.
if you solved it - redo the PR

Reply:
The PR is not ready yet. I just wanted to let you know what the issue was, if you were curious.